src/Form/AkamaiCacheControlForm.php

```diff
@@ -11,9 +11,7 @@
 use Drupal\Core\Form\FormStateInterface;
 
 /**
```

Comment:
Add what exactly? Now seems as good of a time as ever. If we move this functionality into purge, shouldn't it have a plugin system such that something like Akamai could register itself?